firebase/firestore/tests/rule-test.spec.js

```diff
@@ -15,6 +15,7 @@ describe("Rule tests", () => {
 
 
         // 아무것도 업데이트 안해도 성공
+        // ... up test...
```

Comment:
채팅 시작.

Reply:
Make a suggestion...
```suggestion
        // ... up test...
```